department-of-veterans-affairs/caseflow
# Disallow judge removal from their own teams

Resolves #12492 

### Description
Displays an error when a user tries to remove a judge from their team

### Acceptance Criteria
- [ ] Admins cannot remove a judge from their judge team

### Testing Plan
1. Sign in a TEAM ADMIN
1. Go to /organizations/bvaaabshire/users
1. Press "Remove user" next to "Judge with hearings and cases (BVAAABSHIRE)"
1. Ensure an error is displayed

### User Facing Changes
<img width="897" alt="Screen Shot 2019-11-13 at 3 52 37 PM" src="https://user-images.githubusercontent.com/45575454/68804086-fc565500-062e-11ea-9442-6bde567731dd.png">

 ### Notes
This can be circumvented by removing admin rights from the judge and then removing them as a user
